src/System.Net.Security/src/project.json

```diff
@@ -3,7 +3,8 @@
     "System.Diagnostics.Contracts": "4.0.0-*",
     "System.Threading": "4.0.0-*",
     "System.Net.Primitives": "4.0.10-beta-*",
-    "System.Security.Cryptography.OpenSsl": "4.0.0-beta-*"
+    "System.Security.Cryptography.OpenSsl": "4.0.0-beta-*",
```

Comment:
@bartonjs @ericstj PTAL I thought that all dependencies should be platform-independent (the reason why we removed Win32Exception from all contracts). I don't think the Win32 implementation should take a dependency on OpenSSL.